### DISCUSSION TITLE
Search location within boundaries (radius)

### DISCUSSION THREAD

**Discussion by netdesignr** [uber/react-map-gl#2024] [Q&A]
Hi,

I have got an endpoint that has got some predefined locations. I want to allow the user to search for location and set the radius for the search and if there's a possible match return values from the endpoint, will `react-map-gl` allow for searching location with radius set to Miles or KM?

Thanks a lot!

**Comment by Pessimistress** [comment]
This is something you should implement in the backend. Otherwise check out [turf.js](http://turfjs.org) for calculating distance between geolocations.
